### ISSUE TITLE
Old jekyll requirement

### ISSUE THREAD

**Issue by makew0rld** [manunamz/jekyll-wikilinks#39]
https://github.com/manunamz/jekyll-wikilinks/blob/e3270a8adc9d07a4de90550bd846dd3e2ca08b75/jekyll-wikilinks.gemspec#L31

Is this requirement still needed? This plugin won't work with the latest Jekyll because of this line.

**Comment by DevlopersField** [comment]
You can specify the exact version in your Gemfile like this:

`gem 'jekyll', '4.2.0'`

Or install a specific version from the command line:

`gem install jekyll -v 4.2.0`

**Comment by makew0rld** [comment]
Yes, I know. But ideally this plugin would be updated so that later versions of Jekyll could be used.
